cmd/sebak/cmd/wallet/pay.go

```diff
@@ -0,0 +1,103 @@
+package wallet
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"boscoin.io/sebak/cmd/sebak/common"
+
+	"boscoin.io/sebak/lib"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/network"
+	"github.com/stellar/go/keypair"
+
+	"log"
+)
+
+var (
+	flagNetworkID string = sebakcommon.GetENVValue("SEBAK_NETWORK_ID", "")
+	PayCmd        *cobra.Command
+	flagEndpoint  string
+)
+
+func init() {
+	PayCmd = &cobra.Command{
+		Use:   "pay <amount> <sender private key> <receiver pubkey>",
+		Short: "Send <amount> BOSCoin from one wallet to another",
+		Args:  cobra.ExactArgs(3),
+		Run: func(c *cobra.Command, args []string) {
+			var err error
+			var amount sebak.Amount
+			var sender keypair.KP
+			var receiver keypair.KP
+			var endpoint *sebakcommon.Endpoint
+
+			// Amount
+			if amount, err = common.ParseAmountFromString(args[0]); err != nil {
+				common.PrintFlagsError(c, "<amount>", err)
+			}
+
+			// Sender's private key
+			if sender, err = keypair.Parse(args[1]); err != nil {
+				common.PrintFlagsError(c, "<sender private key>", err)
+			} else if _, err = sender.Sign([]byte("witness")); err != nil {
```

Comment:
Oh, very interesting part. This kind of approach, I love it. :) Anyway in terms of information, this can be like this,
```go
} else if _, ok = sender.(*keypair.Full); !ok {
```
`keypair.Parse()` returns `keypair.KP` interface, but we need `*keypair.Full`.